CONTRIBUTING.md

```diff
@@ -103,3 +102,58 @@ developers.
 
   More information about tool development can be found [on the
   wiki](https://wiki.galaxyproject.org/Develop).
+
+## Handling Pull Requests
+
+Everyone is encouraged to express opinions and issue non-binding votes on pull
+requests, but only members of the *committers* group may issue binding votes
+on pull requests. Information on the *committers* group can be found in the
+[organization document](https://github.com/galaxyproject/galaxy/blob/dev/doc/source/project/organization.rst)
+describing governance of the core Galaxy code base.
+
+Votes on pull requests should take the form of
+[+1, 0, -1, and fractions](http://www.apache.org/foundation/voting.html)
+as outlined by the Apache Foundation.
+
+Pull requests modifying pre-existing releases should be restricted to bug fixes
+and require at least 2 *+1* binding votes from someone other than the author of
+the pull request with no *-1* binding votes.
+
+Any pull request changing or clarifying the procedures governing this
+repository must remain open for at least 4 days and require at least 3 *+1*
```

Comment:
I would perhaps remove consideration of weekends/holidays and instead make it at least 8 days (week +1 day), specifically stated in hours ( = 192 hours).

I would also say 3 +1 is far too few, as that is < 16% of committers. Something like ceil( max( 3, 51% of committers) ) voting +1, with no -1; although 51% (currently 10 votes) may still be too small.



IMHO, changing formalized well-established procedures is something that should happen exceptionally rarely, and only when overwhelming cause dictates.